docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1462,6 +1468,11 @@ It is not allowed to specify the property `shadowEntityClass` in `@PiggybackShad
 when the source shadow variable is `@CascadingUpdateShadowVariable`.
 ====
 
+[WARNING]
+====
+The user's logic is responsible for defining the order in which each variable is updated.
```

Comment:
I don't understand what this is trying to say. Maybe it needs a bit of context?

Not sure if it's a warning. If it has more text explaining why this is bad, then I say it could be a warning.

Reply:
I believe it caused more confusion and I will remove it.